### PR TITLE
schemes: init at 0.2.0

### PR DESCRIPTION
**Pull request by 0xMRTT**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by 0xMRTT** [comment]
@ofborg eval

**Comment by AndersonTorres** [comment]
Squash all commits into one.

File: pkgs/applications/misc/schemes/default.nix [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    gtk4
+    libadwaita
+    gtksourceview5
+    libpanel
+  ];
+
+  meta = with lib; {
+    description = "Edit GtkSourceView style-schemes for an application or platform";
+    homepage = "https://gitlab.gnome.org/chergert/schemes";
+    mainProgram = "schemes";
```
Comment (AndersonTorres) [inline pkgs/applications/misc/schemes/default.nix:50]: Is this line necessary? mainProgram defaults to pname, afaicr.

File: pkgs/applications/misc/schemes/default.nix [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, meson
```
Comment (AndersonTorres) [inline pkgs/applications/misc/schemes/default.nix:4]: As a non-binding advice, put this list in ASCIIbetical order (after fetchFromGitLab).

File: pkgs/applications/misc/schemes/default.nix [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+stdenv.mkDerivation rec {
+  pname = "schemes";
+  version = "0.2.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "chergert";
+    repo = "schemes";
+    rev = version;
+    sha256 = "sha256-XUC24KzZSU4+F2JZMsydukvAwEGdMxCnkPG6QHnCw6w=";
```
Comment (AndersonTorres) [inline pkgs/applications/misc/schemes/default.nix:27]:
```suggestion
    hash = "sha256-XUC24KzZSU4+F2JZMsydukvAwEGdMxCnkPG6QHnCw6w=";
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Comment by 0xMRTT** [comment]
I messed up with my git, i opened a new PR instead

https://github.com/NixOS/nixpkgs/pull/227756

**Pull request closed without merging**